src/seat/keyboard/state.rs

```diff
@@ -210,18 +245,21 @@ impl KbState {
     }
 
     pub(crate) fn new() -> Result<KbState, Error> {
-        let xkbh = match ffi::XKBCOMMON_OPTION.as_ref() {
-            Some(h) => h,
-            None => return Err(Error::XKBNotFound),
+        #[cfg(feature = "dlopen")]
+        {
+            if ffi::XKBCOMMON_OPTION.as_ref().is_none() {
+                return Err(Error::XKBNotFound);
+            }
+        }
+        let context = unsafe {
+            ffi_dispatch!(XKBH, xkb_context_new, ffi::xkb_context_flags::XKB_CONTEXT_NO_FLAGS)
         };
```

Comment:
Was there a particular need for renaming this variable?

Reply:
Yes. Otherwise it shadows the `keyboard::ffi::xkb_context` function when the `dlopen` feature is disabled. I suppose this could also be fixed by a more careful use statement but it would get unwieldy considering the large number of functions from the ffi module used in this file. I was just following the instructions in the dlib readme that said to do something like this:

```rust
#[cfg(feature = "dlopen-foo")]
use ffi::FOO_STATIC;
#[cfg(not(feature = "dlopen-foo"))]
use ffi::*;
```

Reply:
Oh, ok, alright.